app/models/supervisor_volunteer.rb

```diff
@@ -24,8 +24,8 @@ def ensure_supervisor_and_volunteer_belong_to_same_casa_org
 #  is_active     :boolean          default(TRUE)
 #  created_at    :datetime         not null
 #  updated_at    :datetime         not null
-#  supervisor_id :integer          not null
-#  volunteer_id  :integer          not null
+#  supervisor_id :bigint           not null
```

Comment:
so confused why they changed from int to bigint.

Reply:
I think it's a local version difference with postgres maybe - bigint is arguably more correct.